subprojects/docs/src/main/resources/header.html

```diff
@@ -95,92 +95,78 @@
-                <ul id="customizing-execution">
-                    <li><a href="/userguide/build_environment.html">Configuring the Build Environment</a></li>
-                    <li><a href="/userguide/gradle_daemon.html">Configuring the Gradle Daemon</a></li>
-                    <li><a href="/userguide/init_scripts.html">Initialization Scripts</a></li>
-                </ul>
-            </li>
-
-            <li><a href="/userguide/gradle_wrapper.html">Provisioning Gradle (Wrapper)</a></li>
-            <li><a href="/userguide/troubleshooting.html">Troubleshooting</a></li>
+            <li><a href="/userguide/composite_builds.html">Integrating Separate Gradle Builds (Composite Builds)</a></li>
+            <li><a href="/userguide/troubleshooting.html">Troubleshooting builds</a></li>
         </ul>
 
         <h3 id="authoring-gradle-builds">Authoring Gradle Builds</h3>
         <ul>
-            <li><a href="/userguide/groovy_build_script_primer.html">Groovy DSL Primer</a></li>
-            <li><a href="/userguide/kotlin_dsl.html">Kotlin DSL Primer</a></li>
-            <li><a href="/userguide/authoring_maintainable_build_scripts.html">Authoring Maintainable Builds</a></li>
-            <li><a class="nav-dropdown" data-toggle="collapse" href="#authoring-build-scripts" aria-expanded="false" aria-controls="authoring-build-scripts">Fundamentals</a>
+            <li><a class="nav-dropdown" data-toggle="collapse" href="#authoring-build-scripts" aria-expanded="false" aria-controls="authoring-build-scripts">Learning the Basics</a>
                 <ul id="authoring-build-scripts">
                     <li><a href="/userguide/tutorial_using_tasks.html">Introducing the Basics of Build Scripts</a></li>
                     <li><a href="/userguide/more_about_tasks.html">Working with Tasks</a></li>
                     <li><a href="/userguide/writing_build_scripts.html">Learning More About Build Scripts</a></li>
                     <li><a href="/userguide/working_with_files.html">Working with Files</a></li>
                     <li><a href="/userguide/custom_tasks.html">Creating Custom Task Types</a></li>
                     <li><a href="/userguide/plugins.html">Using Gradle Plugins</a></li>
-                    <li><a href="/userguide/standard_plugins.html">The Standard Gradle Plugins</a></li>
                     <li><a href="/userguide/build_lifecycle.html">Understanding the Build Lifecycle</a></li>
                     <li><a href="/userguide/logging.html">Working with Logging</a></li>
                     <li><a href="/userguide/multi_project_builds.html">Configuring Multi-Project Builds</a></li>
                 </ul>
             </li>
+            <li><a href="/userguide/organizing_gradle_projects.html">Organizing Build Logic</a></li>
+            <li><a href="/userguide/authoring_maintainable_build_scripts.html">Authoring Maintainable Builds</a></li>
```

Comment:
I've moved this down because I think it's important that the introductory content appears first. And this chapter kind of depends on the information presented in the introductory chapters.